nservicebus/upgrades/7to8/index.md

```diff
@@ -53,6 +53,88 @@ The following provider packages will no longer be provided:
 * [Log4net](/nservicebus/logging/log4net.md)
 * [NLog](/nservicebus/logging/nlog.md)
 
+## CancellationToken support
```

Comment:
@DavidBoike @adamralph I wonder if we should move most of the content in this section to the new page I created and just make this a short paragraph linking to it?